data/src/main/java/org/akvo/flow/data/net/FlowApiService.java

```diff
@@ -20,6 +20,16 @@
 
 package org.akvo.flow.data.net;
 
-public interface FlowApiService {
+import org.akvo.flow.data.entity.ApiLocaleResult;
 
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.Url;
+import rx.Observable;
+
+interface FlowApiService {
+
+    @GET
```

Comment:
Yes, this is how it is normally done but our url is a bit odd as some parts are encoded. Will try something tonight and see if it works